homeassistant/components/goalzero/entity.py

```diff
@@ -0,0 +1,48 @@
+"""Entity representing a Goal Zero Yeti device."""
+
+from goalzero import Yeti
+
+from homeassistant.const import ATTR_MODEL, CONF_NAME
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo, EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import ATTRIBUTION, DOMAIN, MANUFACTURER
+from .coordinator import GoalZeroDataUpdateCoordinator
+
+
+class GoalZeroEntity(CoordinatorEntity[GoalZeroDataUpdateCoordinator]):
+    """Representation of a Goal Zero Yeti entity."""
+
+    _attr_attribution = ATTRIBUTION
+
+    def __init__(
+        self,
+        coordinator: GoalZeroDataUpdateCoordinator,
+        description: EntityDescription,
+    ) -> None:
+        """Initialize a Goal Zero Yeti entity."""
+        super().__init__(coordinator)
+        self.coordinator = coordinator
```

Comment:
We can remove this line. The coordinator attribute is set by the coordinator entity.